broker-cli/commands/orderbook.js

```diff
@@ -107,11 +106,24 @@ function createUI (market, asks, bids) {
  * @param {Logger} logger
  */
 async function orderbook (args, opts, logger) {
-  const { market, rpcAddress } = opts
+  const { market, rpcAddress, json, stream } = opts
   const request = { market }
 
   try {
     const brokerDaemonClient = new BrokerDaemonClient(rpcAddress)
+
+    if (!stream) {
```

Comment:
if the client uses `--json` but not `--stream` we should show an error, otherwise they will pass `--json` but it will have no effect.

I also think that streaming should be the default, so the option is better as `--no-stream` or `--static`

Reply:
Using `--no-stream` as option and returning failure if json and is passed without --no-stream